extensions/amp-story-360/validator-amp-story-360.protoascii

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-story-360
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-story-360"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
```

Comment:
Can you add `mandatory_ancestor: "AMP-STORY"` to make it invalid for any AMP document that's not a story? You'll have to update your test cases, sorry about that :(

Reply:
What is the point of enforcing this ? Sounds rather arbitrary.

Reply:
Making it available for all AMP documents and not just stories would require a lot more thinking and user controls. One example that comes to mind: video have play/pause/audio controls, but you still want to be able to pan in the video when the option is provided, how do you handle that?
Until we think of all these edge cases and talk to the UI team who owns these components, we should not launch something behind their back.

Reply:
Done. Note that it makes the test in amp-story and amp-story-360 basically the same.